Packs/Hoxhunt/Playbooks/Hox-Enrich-Incident_README.md

```diff
@@ -0,0 +1,37 @@
+Enriches the incidents pulled from Hoxhunt with additional information.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+This playbook does not use any integrations.
+
+### Scripts
+
+* IsIntegrationAvailable
+
+### Commands
+
+* rasterize
+
+## Playbook Inputs
+
+---
+There are no inputs for this playbook.
+
+## Playbook Outputs
+
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+
+---
+
+![Hoxhunt - Enrich Incident](../doc_files/Hoxhunt_-_Enrich_Incident.png)
```

Comment:
The image file doesn't exist under the doc_files dir and hasn't the same file name.
Please move out the image file to the doc_files dir and fix the name and make sure you can see the image in the playbook readme in XSOAR.